src/slic3r/GUI/WipeTowerDialog.cpp

```diff
@@ -581,21 +581,19 @@ WipingPanel::WipingPanel(wxWindow* parent, const std::vector<float>& matrix, con
 
         auto on_apply_text_modify = [this](wxEvent& e) {
             wxString str = m_flush_multiplier_ebox->GetValue();
-            str.Replace(",", ".");
-            double  multiplier = 1.f;
-            if (str.ToDouble(&multiplier)) {
-                if (multiplier < g_min_flush_multiplier || multiplier > g_max_flush_multiplier) {
-                    str = wxString::Format(("%.2f"), multiplier < g_min_flush_multiplier ? g_min_flush_multiplier : g_max_flush_multiplier);
-                    m_flush_multiplier_ebox->SetValue(str);
-                    MessageDialog dlg(nullptr,
-                        wxString::Format(_L("The multiplier should be in range [%.2f, %.2f]."), g_min_flush_multiplier, g_max_flush_multiplier),
-                        _L("Warning"), wxICON_WARNING | wxOK);
-                    dlg.ShowModal();
-                }
-                for (unsigned int i = 0; i < m_number_of_extruders; ++i) {
-                    for (unsigned int j = 0; j < m_number_of_extruders; ++j) {
-                        edit_boxes[i][j]->SetValue(to_string(int(m_matrix[m_number_of_extruders * j + i] * multiplier)));
-                    }
+            str.Replace(".", ",");
```

Comment:
Will this cause problems when using languages that don't use commas as decimal points?

Reply:
The PR came from Github of Bambu Studio to correct the multiplier that was only working with decimal numbers in English, but by correcting this they also changed the dot to comma (probably because most languages use comma to decimal), for this I also put the Substitute function for languages that use dot as decimal.

Reply:
I'll record the screen in all version